packages/rule-no-broken-links/README.md

```diff
@@ -0,0 +1,64 @@
+# Check for broken links
```

Comment:
Add `(`@sonarwhal/rule-broken-links`)` back. I think @alrra was taking about the first part of the title :)